docs/scenario_setup.rst

```diff
@@ -0,0 +1,93 @@
+.. _scenario_setup_label:
```

Comment:
Please rename label and file according to new heading in line 4.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/26f19941b96e9dd91c70a010339dfba5e315bfc6.